src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IParameterReferenceExpression.vb

```diff
@@ -630,11 +630,7 @@ End Class]]>.Value
 
             Dim expectedOperationTree = <![CDATA[
 IInvalidStatement (OperationKind.InvalidStatement, IsInvalid) (Syntax: 'Case x')
-  Children(1):
```

Comment:
I think strip child for all invalid, but just for this particular case. where case statement are wield in VB currently. once we fix IOperation tree around case block/statement, it should have those child again.